cla_backend/libs/eligibility_calculator/calculator.py

```diff
@@ -441,6 +451,7 @@ def _translate_case(case_data, submission_date=None):
         if hasattr(case_data, "facts"):
             request_data['applicant'].update(
                 EligibilityChecker._translate_applicant_data(submission_date, case_data.facts))
+            request_data["assessment"].update(translate_assessment(case_data.facts))
```

Comment:
think there's a place in the code which creates the 'assessment' key - this looks.a bit out of place